src/lib/common/globals.h

```diff
@@ -221,6 +225,9 @@ extern bool               disableCusNotif;
 
 extern bool               insecureNotif;
 
+#ifdef PARANOID_JSON_INDENT
+extern bool               paranoidV1Indent;
```

Comment:
If this var is to be maintained (I kind of doubt it), same comment as before, look for a name in the line of 'human readable format', which is what the whole thing is about

Reply:
I will not be mantained. See https://github.com/telefonicaid/fiware-orion/issues/2760#issuecomment-337334053.

Reply:
Ok

NTC